tutorials/epochs/make_fixed_length_epochs.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
```

Comment:
You need to name the file `plot_*` to get it to be rendered by CircleCI:

https://26238-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/epochs/make_fixed_length_epochs.html#sphx-glr-auto-tutorials-epochs-make-fixed-length-epochs-py

Looks already quite polished so it will be nice to see the CircleCI result!